multiversx_sdk/core/transactions_factories/token_management_transactions_factory.py

```diff
@@ -654,10 +709,10 @@ def create_transaction_for_local_minting(
         token_identifier: str,
         supply_to_mint: int
     ) -> Transaction:
-        parts: list[str] = [
+        parts = [
             "ESDTLocalMint",
-            arg_to_string(token_identifier),
-            arg_to_string(supply_to_mint)
+            self.serializer.serialize([StringValue(token_identifier)]),
```

Comment:
Directly call `hex()`? Can also stay as it is.

Reply:
No need to call `hex()`, the result is already a `hex string`. Or do you mean to do:
```py
token_identifier.encode().hex()
```
?